### DISCUSSION TITLE
App not starting Windows 10

### DISCUSSION THREAD

**Discussion by labsektor** [TumblThreeApp/TumblThree#358] [Q&A (Question & Answer)]
When I'm starting app (doubleclick or run as Admin) i see only loading circle at cursor and nothing happend.

**To Reproduce**
Run app

**Desktop (please complete the following information):**
Checkend on TumblThree 1.4 and 1.3
Version 1.0 is working

 - Windows 10

Already tried disabling antivirus and reinstall .Net

**Comment by thomas694** [comment]
Please provide us more information. Which version 1.0 exactly do you mean?
You can look in the TumblThree log file for a hint. The log file should be in %LOCALAPPDATA%\TumblThree or inside the program folder for portable mode.
Also there could be a hint in the windows event log why the app fails to start on your system.

**Comment by labsektor** [comment]
Got it in windows log
Nazwa dziennika:Application
Źródło:        .NET Runtime
Data:          06.04.2021 20:06:44
Identyfikator zdarzenia:1026
Kategoria zadania:Brak
Poziom:        Błędy
Słowa kluczowe:Klasyczny
Użytkownik:    Nie dotyczy
Komputer:      DESKTOP-51AM4AC
Opis:
Aplikacja: TumblThree.exe
Wersja architektury: v4.0.30319
Opis: proces został przerwany z powodu nieobsłużonego wyjątku.
Informacje o wyjątku: System.NullReferenceException
   w TumblThree.Domain.Logger.Log(System.Diagnostics.TraceEventType, System.String, System.Object[])
   w TumblThree.Applications.Controllers.ModuleController.LoadSettings[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.String)
   w TumblThree.Applications.Controllers.ModuleController.Initialize()
   w TumblThree.Presentation.App.OnStartup(System.Windows.StartupEventArgs)
   w System.Windows.Application.<.ctor>b__1_0(System.Object)
   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   w System.Windows.Threading.DispatcherOperation.InvokeImpl()
   w MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   w System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   w System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   w System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   w MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   w System.Windows.Threading.DispatcherOperation.Invoke()
   w System.Windows.Threading.Dispatcher.ProcessQueue()
   w System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   w MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   w MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   w System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   w MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   w MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   w System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   w System.Windows.Application.RunDispatcher(System.Object)
   w System.Windows.Application.RunInternal(System.Windows.Window)
   w TumblThree.Presentation.App.Main()


Kod XML zdarzenia:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2021-04-06T18:06:44.4979484Z" />
    <EventRecordID>56809</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>DESKTOP-51AM4AC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Aplikacja: TumblThree.exe
Wersja architektury: v4.0.30319
Opis: proces został przerwany z powodu nieobsłużonego wyjątku.
Informacje o wyjątku: System.NullReferenceException
   w TumblThree.Domain.Logger.Log(System.Diagnostics.TraceEventType, System.String, System.Object[])
   w TumblThree.Applications.Controllers.ModuleController.LoadSettings[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.String)
   w TumblThree.Applications.Controllers.ModuleController.Initialize()
   w TumblThree.Presentation.App.OnStartup(System.Windows.StartupEventArgs)
   w System.Windows.Application.&lt;.ctor&gt;b__1_0(System.Object)
   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   w System.Windows.Threading.DispatcherOperation.InvokeImpl()
   w MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   w System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   w System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   w System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   w MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   w System.Windows.Threading.DispatcherOperation.Invoke()
   w System.Windows.Threading.Dispatcher.ProcessQueue()
   w System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   w MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   w MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   w System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   w System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   w System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   w MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   w MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   w System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   w System.Windows.Application.RunDispatcher(System.Object)
   w System.Windows.Application.RunInternal(System.Windows.Window)
   w TumblThree.Presentation.App.Main()

</Data>
  </EventData>
</Event>

-----------------------

Nazwa dziennika:Application
Źródło:        Application Error
Data:          06.04.2021 20:06:44
Identyfikator zdarzenia:1000
Kategoria zadania:(100)
Poziom:        Błędy
Słowa kluczowe:Klasyczny
Użytkownik:    Nie dotyczy
Komputer:      DESKTOP-51AM4AC
Opis:
Nazwa aplikacji powodującej błąd: TumblThree.exe, wersja: 1.4.0.112, sygnatura czasowa: 0x6066231b
Nazwa modułu powodującego błąd: unknown, wersja: 0.0.0.0, sygnatura czasowa: 0x00000000
Kod wyjątku: 0xc0000005
Przesunięcie błędu: 0x00007ffe9d838aab
Identyfikator procesu powodującego błąd: 0x3078
Godzina uruchomienia aplikacji powodującej błąd: 0x01d72b0f9a26221e
Ścieżka aplikacji powodującej błąd: C:\Users\neon8\Downloads\TumblThree-v1.4.0-x64-Application\TumblThree\TumblThree.exe
Ścieżka modułu powodującego błąd: unknown
Identyfikator raportu: e3e769e3-d90e-47a5-a882-3efb1150297c
Pełna nazwa pakietu powodującego błąd: 
Identyfikator aplikacji względem pakietu powodującego błąd: 
Kod XML zdarzenia:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" />
    <EventID Qualifiers="0">1000</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>100</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2021-04-06T18:06:44.6149750Z" />
    <EventRecordID>56810</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>DESKTOP-51AM4AC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>TumblThree.exe</Data>
    <Data>1.4.0.112</Data>
    <Data>6066231b</Data>
    <Data>unknown</Data>
    <Data>0.0.0.0</Data>
    <Data>00000000</Data>
    <Data>c0000005</Data>
    <Data>00007ffe9d838aab</Data>
    <Data>3078</Data>
    <Data>01d72b0f9a26221e</Data>
    <Data>C:\Users\neon8\Downloads\TumblThree-v1.4.0-x64-Application\TumblThree\TumblThree.exe</Data>
    <Data>unknown</Data>
    <Data>e3e769e3-d90e-47a5-a882-3efb1150297c</Data>
    <Data>
    </Data>
    <Data>
    </Data>
  </EventData>
</Event>

**Comment by thomas694** [comment]
There seems something wrong with your app setting files. Make a backup of the files in %LOCALAPPDATA%\TumblThree\Settings, then delete them and try to run TumblThree again. If you like and want to find out which one was making the trouble, you can delete them one by one and always check if it already runs again.

**Comment by labsektor** [comment]
it helped
thank you very much
